regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -56,17 +82,23 @@ var PreambleView = ChildView.extend({
 
   render: function() {
     ChildView.prototype.render.apply(this, arguments);
+
     this.$read = this.$el.find('#preamble-read');
     this.$write = this.$el.find('#preamble-write');
-    var section = this.$read.closest('section').attr('id');
-    var docId = section.split('-')[0];
+
+    this.currentSectionId = this.$read.closest('section').attr('id');
+    this.docId = this.currentSectionId.split('-')[0];
+
+    this.preambleHeadView = new PreambleHeadView();
```

Comment:
Curious: right now `PreambleView` holds a reference to `PreambleHeadView` (which it doesn't ever use), yet `PreambleHeadView` must reach into `PreambleView` via jquery (e.g. `$('#preamble-write').is(':visible')`) or through events. Would it make sense to couple these views _tighter_ here (given how related they are)?

For example, instead of having `PreambleHeadView` use jquery to find the links, should they just be passed in to its constructor? Instead of triggering a new event on tab open, should `PreambleView` call `this.preambleHeadView.writeTabOpen()` directly?